kubeflow/trainer/rhai/traininghub.py

```
@@ -146,12 +661,82 @@ def _compose_exec_script(func_code: str, func_file: str) -> str:
+
+# =============================================================================
+# USER TRAINING CODE STARTS BELOW
+# =============================================================================
+
+"""
+
+    return wrapper
+
+
 def get_trainer_cr_from_training_hub_trainer(
     runtime: types.Runtime,
```

Ruff (0.14.8)

Unused function argument: `runtime`

(ARG001)